packages/shopify-app-session-storage-mysql/src/mysql.ts

```diff
@@ -1,12 +1,27 @@
 import mysql from 'mysql2/promise';
 import {Session} from '@shopify/shopify-api';
-import {SessionStorage} from '@shopify/shopify-app-session-storage';
+import {
+  SessionStorage,
+  SessionStorageMigratorOptions,
+  SessionStorageMigrator,
+  RdbmsSessionStorageMigrator,
+  RdbmsSessionStorageOptions,
+} from '@shopify/shopify-app-session-storage';
+
+import {migrationMap} from './migrations';
+import {MySqlEngine} from './mysql-engine';
+
+export interface MySQLSessionStorageOptions
+  extends RdbmsSessionStorageOptions {}
 
-export interface MySQLSessionStorageOptions {
-  sessionTableName: string;
-}
 const defaultMySQLSessionStorageOptions: MySQLSessionStorageOptions = {
   sessionTableName: 'shopify_sessions',
+  sqlArgumentPlaceholder: '?',
```

Comment:
I feel like this should be a static parameter of the mysql implementation, and we would just have a generic `placeholders()` method called by the abstract class whenever `query` is called. So something like

```ts
this.query(query, [1, 2]);

function query(query, args) {
  this.queryImplementation(query, this.escapeArgsImplementation(args));
}
```

does that make sense? 😄